### ISSUE TITLE
bigip_configsync_action fails when sync status is red

### ISSUE THREAD

**Issue by techoover** [F5Networks/f5-ansible#1472]
##### ISSUE TYPE
 - Bug Report

bigip_configsync_action works fine if changes were made only on single member of the HA.
if changes were made on both (sync status is red) it fails

##### COMPONENT NAME
bigip_configsync_

##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.8.3
```

##### PYTHON VERSION

```
$ python -V
Python 2.7.15+
```

##### BIGIP VERSION

```
# tmsh show sys version

Sys::Version
Main Package
  Product     BIG-IP
  Version     14.1.0.2
  Build       0.0.4
  Edition     Point Release 2
  Date        Thu Feb 14 20:01:44 PST 2019

```

##### ROLE VERSION
no roles used
```

```

##### CONFIGURATION
none

##### OS / ENVIRONMENT
Ubuntu 18.04.2 LTS

##### STEPS TO REPRODUCE

```
- name: Sync 
  bigip_configsync_action:
      device_group: sync-failover
      sync_device_to_group: yes
      provider: "{{provider}}"
  when: hostvars[inventory_hostname].bigip1 is defined

```

##### EXPECTED RESULTS
normal sync

##### ACTUAL RESULTS

```
TASK [Sync ] ***********************************************************************************************************************************************************************************
task path: /home/username/ansible-f5/f5-do/4-save-config-bigip.yaml:32
<brlab1-sfn.merck.com> connection transport is rest
<brlab1-sfn.merck.com> ESTABLISH LOCAL CONNECTION FOR USER: username
<brlab1-sfn.merck.com> EXEC /bin/sh -c 'echo ~username && sleep 0'
<brlab1-sfn.merck.com> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535 `" && echo ansible-tmp-1566983512.08-208776211531535="` echo /home/username/.ansible/tmp/asible-tmp-1566983512.08-208776211531535 `" ) && sleep 0'
skipping: [brlab2-sfn.merck.com] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}
Using module file /home/username/.local/lib/python2.7/site-packages/ansible/modules/network/f5/bigip_configsync_action.py
<brlab1-sfn.merck.com> PUT /home/username/.ansible/tmp/ansible-local-20122YWD3Is/tmpN6o64_ TO /home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py
<brlab1-sfn.merck.com> EXEC /bin/sh -c 'chmod u+x /home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/ /home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py && leep 0'
<brlab1-sfn.merck.com> EXEC /bin/sh -c '/usr/bin/python /home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py && sleep 0'
<brlab1-sfn.merck.com> EXEC /bin/sh -c 'rm -f -r /home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py", line 114, in <module>
    _ansiballz_main()
  File "/home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py", line 106, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py", line 49, in invoke_module
    imp.load_module('__main__', mod, module, MOD_DESC)
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 427, in <module>
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 420, in main
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 209, in exec_module
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 231, in present
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 256, in execute
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 313, in _wait_for_sync
  File "/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py", line 355, in _get_details_from_resource
NameError: global name 'Tree' is not defined

fatal: [brlab1-sfn.merck.com]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"/home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py\", line 114, in <module>\n    _ansiballz_main()\n  File "/home/username/.ansible/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py\", line 106, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/username/.ansibe/tmp/ansible-tmp-1566983512.08-208776211531535/AnsiballZ_bigip_configsync_action.py\", line 49, in invoke_module\n    imp.load_module('__main__', mod, module, MOD_DESC)\n  File \"/tmp/ansible_bigip_configsync_action_payload_NoycA/__main__.py\", line 427, in <module>\n  File \"/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py\", line 420, in main\n  File \"/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py\", line 209, in exec_odule\n  File \"/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py\", line 231, in present\n  File \"/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py\", line 256, in execute\n  File \"/tmp/ansible_bigi_configsync_action_payload_NoycNA/__main__.py\", line 313, in _wait_for_sync\n  File \"/tmp/ansible_bigip_configsync_action_payload_NoycNA/__main__.py\", line 355, in _get_details_from_resource\nNameError: global name 'Tree' is nt defined\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}

```

**Comment by techoover** [comment]
STEPS TO REPRODUCE disappeared. 
Make changes on two members of sync group individually and try to sync device to group.

**Comment by wojtek0806** [comment]
https://github.com/F5Networks/f5-ansible/issues/1361duplicate already fixed for 2.9
